app/src/util/tools.ts

```diff
@@ -758,7 +758,7 @@ export const reverseArray = (array: any[]): any[] => {
   return newArray
 }
 
-export const getBlockExplorerURL = (networkId: number, txHash: string): string => {
+export const getTxHashBlockExplorerURL = (networkId: number, txHash: string): string => {
```

Comment:
Don't know if it's scope of this issue but i would add a suggestion that all etherscan links are stored in networks and that we make a function that fetches it based on network like we did for bunch of stuff....like for example getChainSpecifiAlternativeUrls? @KadenZipfel @hexyls what do you think?

Reply:
Good call! I've moved it to the networks file